mne_neuron/sim.py

```diff
@@ -0,0 +1,17 @@
+# import NEURON module
+from neuron import h
```

Comment:
I wonder if we should call the file `utils.py` or `config.py`. The name doesn't speak to me at the moment. sim = simulate?

Reply:
or maybe `parallel.py`? Since it is managing the parallel context